nautobot/core/apps/__init__.py

```diff
@@ -691,6 +691,66 @@ def convert_biginteger(field, registry=None):  # pylint: disable=redefined-outer
 
         monkey_mix(TaggableManager, mixins.TaggableManagerMonkeyMixin)
 
+        from django.db.models.sql.query import Query
+        from django.db.models.expressions import Ref
+
+        def set_values(self, fields):
```

Comment:
Do we need to forklift the whole method, or could we instead just do something like:

```
def set_values(self, fields):
    if fields:
        for field in fields:
            self.check_alias(field)
    super().set_values(fields)
```

Reply:
I was running into this when I tried this method:
```
Traceback (most recent call last):
  File "/source/nautobot/core/cli.py", line 182, in <module>
    main()
  File "/source/nautobot/core/cli.py", line 54, in main
    run_app(
  File "/source/nautobot/core/runner/runner.py", line 297, in run_app
    management.execute_from_command_line([runner_name, command] + command_args)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 395, in execute
    django.setup()
  File "/usr/local/lib/python3.11/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.11/site-packages/django/apps/registry.py", line 122, in populate
    app_config.ready()
  File "/source/nautobot/circuits/apps.py", line 9, in ready
    super().ready()
  File "/source/nautobot/core/apps/__init__.py", line 50, in ready
    register_menu_items(menu_items)
  File "/source/nautobot/core/apps/__init__.py", line 83, in register_menu_items
    item.link = reverse(item.link, args=item.args, kwargs=item.kwargs)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/urls/base.py", line 54, in reverse
    app_list = resolver.app_dict[ns]
               ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/urls/resolvers.py", line 530, in app_dict
    self._populate()
  File "/usr/local/lib/python3.11/site-packages/django/urls/resolvers.py", line 464, in _populate
    for url_pattern in reversed(self.url_patterns):
                                ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/urls/resolvers.py", line 602, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
                       ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/urls/resolvers.py", line 595, in urlconf_module
    return import_module(self.urlconf_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/source/nautobot/core/urls.py", line 6, in <module>
    from nautobot.core.views import (
  File "/source/nautobot/core/views/__init__.py", line 26, in <module>
    from nautobot.core.constants import SEARCH_MAX_RESULTS, SEARCH_TYPES
  File "/source/nautobot/core/constants.py", line 66, in <module>
    "queryset": Provider.objects.annotate(count_circuits=count_related(Circuit, "provider")),
                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/source/nautobot/utilities/utils.py", line 180, in count_related
    model.objects.filter(**{field: OuterRef("pk")}).order_by().values(field).annotate(c=Count("*")).values("c")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 840, in values
    clone = self._values(*fields, **expressions)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 835, in _values
    clone.query.set_values(fields)
  File "/source/nautobot/core/apps/__init__.py", line 700, in set_values
    super().set_values(fields)
    ^^^^^^^
TypeError: super(type, obj): obj must be an instance or subtype of type
```